File: sdk-node/examples/forms/delete-element.ts
/**
 * Delete an element from a form.
 *
 * Run:
 *   npx tsx examples/forms/delete-element.ts
 */

import { ThreeCommon } from '@3common/sdk'

const client = new ThreeCommon({
  apiKey: '3co_your_api_key_here',
})

const { deletedElementId } = await client.forms.deleteElement(
  'frm_replace_with_real_id',
  'elm_replace_with_real_id',
)

console.log(`deleted ${deletedElementId}`)

File: sdk-python/examples/forms/delete_element.py
"""Run with: python examples/forms/delete_element.py"""

from __future__ import annotations

from threecommon import ThreeCommon


def main() -> None:
    with ThreeCommon(api_key="3co_your_api_key_here") as client:
        result = client.forms.delete_element("frm_replace_with_real_id", "elm_replace_with_real_id")
        print(f"deleted element {result.deleted_element_id}")


if __name__ == "__main__":
    main()
